technology-considerations/TDR Export Format Requirements.md

```diff
@@ -0,0 +1,426 @@
+is done the prefix separator should not use a ":" since this is defined
+by BagIt as the delimiter for a property and its value.
+
+### Consignment Audit
+
+Should an audit of the consignment level properties be required the
+following approach may be used as a solution.
+
+Should any of the values change, be added or deleted during the TDR
+process an audit could be created. The creation of a "consignment-audit"
+file containing a simple list of properties to track the values, dates
+and users may suffice.
+
+A simple .csv format for the file is proposed and an example is shown
+below
+
+![](./images/consignment-audit.png)
+
+Initial values of any properties directly related to the consignment and
+submitted by the user could be captured.
+
+### Payload metadata
+
+In addition to consignment and process information the TDR export
+package is proposed to include information around the file metadata and
+possibly an audit of this information.
+
+The following sections provide examples for the additional tag files
+suggested to convey the required information.
+
+#### File Metadata
+
+A tag file named "file-metadata" is proposed to contain the final values
+of each the metadata properties wishing to be captured for each file.
+This file does not limit or define the properties to be captured, other
+than the "file" column. However this approach allows the definition of
+any properties required using column headers.
+
+Whilst the format of the file could be anything we recommend a simple
+.csv structure with a row describing each file in the payload.
+
+![](./images/file-metadata.png)
+
+Some of the information required in the DRI csv schema files is
+available from the bag-info.txt information and is not proposed to be
+repeated in the metadata file. Although it is possible to de-normalise
+some of the information such as body and series and have it included if
+desired.
+
+#### Metadata Audit.
+
+In situations where a single or multiple users have edited the values of
+properties it may be necessary to track or determine who edited what and
+when for the purposes of provenance and accountability.
+
+Should the need arise, it is proposed that a tag file named
+"metadata-audit" is created for the historical audit trail of metadata
+values. A simple csv format for would suffice.
+
+A proposed example is shown below.
+
+![](./images/metadata-audit.png)
+
+The initial values of any property with its initial date may also be
+recorded in this format even if it has not later been updated or
+amended. If a value is deleted, a null is suggested to be shown as the
+value.
+
+#### 
+
+#### Clientside Metadata
+
+To ensure files are uploaded correctly the TDR application will carry
+out nominal client side evaluation on the files. It is recommended that
+this information is also captured as part of the consignment bag payload
+as the tag file "client-metadata".
+
+An example of the proposed format is shown below
+
+![](./images/client-metadata.png)
```

Comment:
The original path will have to be a relative path, not an absolute path as shown here. We can't look up the absolute path on the client side because of browsers' security restrictions.

Reply:
Yes relative is fine and is all that needed so that we can keep the correct folder structure in the bag

Reply:
Is it possible to edit the image? I just don't want to give anyone false hope that we can preserve the full path. :)

Reply:
I've updated the text so that it explicitly states that the full path may not be accessible and the relative path would be fine without causing issues. I hope this is OK